tests/plugins/inlinetoolbar/positioning.js

```diff
@@ -0,0 +1,92 @@
+/* bender-tags: a11ychecker,inlinetoolbar */
+/* bender-ckeditor-plugins: toolbar,link,inlinetoolbar */
+/* bender-include: ../balloonpanel/_helpers/tools.js */
+/* global balloonTestsTools */
+
+( function() {
+	'use strict';
+
+	CKEDITOR.dom.window.prototype.getViewPaneSize = function() {
+		return {
+			width: 1183,
+			height: 1818
+		};
+	};
+
+	bender.editor = {
+		name: 'editor1',
+		config: {
+			allowedContent: true,
+			height: 300,
+			width: 300
+		}
+	};
+
+	var inlineToolbarConfig = {
+			width: 100,
+			height: 200
+		},
+		spy = sinon.spy,
+		tests = {
+			// There's no need to call editor.window.scrollTo, because balloon logic initialy considers only
+			// scrolling of **outer window**. It does not care about editable window, as its scrolling is already
+			// "contained" in result returned by the element.getClientRect() of marker.
+			_getFrameMethodReplaced: false,
+
+			setUp: function() {
+				this.inlineToolbar = new CKEDITOR.ui.inlineToolbar( this.editor, inlineToolbarConfig );
+
+				this.markerElement = this.editor.editable().findOne( '#marker' );
+
+				// Make sure that global window is scrolled down to 0,0.
+				window.scrollTo( 0, 0 );
+
+				if ( !this._getFrameMethodReplaced ) {
+					// The problem is also window.getFrame().getClientRect() as it retursn different results from dashboard and directly.
```

Comment:
Typo retursn -> returns

Reply:
Whole body of this if instruction could be replaced just with `sinon.stub( this.editor.window, 'getFrame', { height: 300, width: 300, left: 1, bottom: 643, right: 301, top: 343 } );`.

It's much more readable.

Reply:
After consultation with @Comandeer, about problems in this solution we decided to use the solution from `ballonpanel test`